pkgs/development/python-modules/gym/default.nix

```
@@ -15,7 +15,7 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "openai";
     repo = pname;
-    rev = "${version}";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```

This is what we're trying to migrate everything to, because it prevents tag/branch confusion.

---

good point!